src/main/java/com/foundations/convertor/model/Conversion/VideoConversion.java

```diff
@@ -92,12 +92,14 @@ public void doConversion(ConversionCriteria criteria){
             FFprobe fpWrapper = new FFprobe(ffProbePath );
             FFmpeg fmWrapper = new FFmpeg(ffMpegPath);
 
-            FFmpegProbeResult in = fpWrapper.probe(criteria.getInputPath());
+            FFmpegProbeResult in = fpWrapper.probe(criteria.getPath());
 
             String out = criteria.getOutputPath();
-            double fps = criteria.getFramesPerSecond() <=0 ? in.getStreams().get(0).r_frame_rate.getNumerator():criteria.getFramesPerSecond();
-            int resWidth = criteria.getResolutionWidth() <=0 ? in.getStreams().get(0).width: criteria.getResolutionWidth();
-            int resHeight = criteria.getResolutionHeight() <=0 ? in.getStreams().get(0).height : criteria.getResolutionHeight();
+            double fps = criteria.getFrameRate() <=0 ?
+                    in.getStreams().get(0).r_frame_rate.getNumerator(): criteria.getFrameRate();
+            String reso[]=criteria.getResolution().split("X");
```

Comment:
variable reso --- maybe would be directly resolution

Reply:
its just to separate the resolution variable into height and width

Reply:
Already changed to resolution